modules/installation-restricted-network-samples.adoc

```diff
@@ -41,8 +71,6 @@ $ oc create configmap registry-config --from-file=$path/ca.crt -n openshift-conf
 $ oc patch image.config.openshift.io/cluster --patch '{"spec":{"additionalTrustedCA":{"name":"registry-config"}}}' --type=merge
```

Comment:
@dmage does this procedure still look correct (between v4.2 and 4.3)?

$ oc create configmap registry-config --from-file=$path/ca.crt -n openshift-config
$ oc patch image.config.openshift.io/cluster --patch '{"spec":{"additionalTrustedCA":{"name":"registry-config"}}}' --type=merge
$ oc patch image.config.openshift.io/cluster --patch '{"spec":{"additionalTrustedCA":{"name":"registry-config"}}}' --type=merge

And is this note still valid:
"This is required because the imagestream import process does not use the mirror or search mechanism at this time." 

Thanks!

Reply:
Almost, creation of this configmap is described at https://docs.openshift.com/container-platform/4.2/registry/configuring-registry-operator.html#registry-operator-config-resources-configmap_configuring-registry-operator

The key should match the registry host name:
```
$ oc create configmap registry-config --from-file=<external_registry_address>=$path/ca.crt -n openshift-config
```

The patch command looks good.

Regarding the note, I guess it's about `samplesRegistry`. In 4.3 the mirror mechanism is implemented, but we should ask QE if it's possible to install the cluster without this step. Perhaps there is something else that requires this step.